packages/web/src/shared/components/form/ComboboxInput.tsx

```diff
@@ -115,22 +147,26 @@ const ComboboxInput = forwardRef<HTMLInputElement, ComboboxProps>(
             </Combobox.Button>
           </div>
           <Combobox.Options className="z-10 absolute min-w-[160px] w-full max-h-80 bg-white overflow-auto mt-1 rounded border border-slate-400 shadow">
-            {filteredItems.map((item) => (
-              <Combobox.Option
-                className="px-3 py-2 ui-active:bg-blue-400"
-                key={item.value}
-                value={item.value}
-              >
-                {item.value === CREATE_TAG ? (
-                  <>
-                    <Icon icon="add" /> "
-                    <span className="italic">{item.label}</span>"
-                  </>
-                ) : (
-                  item.label
-                )}
-              </Combobox.Option>
-            ))}
+            {filteredItems.length > MAX_ITEMS ? (
+              <div className="px-3 py-2">{t('common:too_many_options')}</div>
+            ) : (
```

Comment:
Rather than showing too many options, what do you think about just truncating to 100 items or so in a scrollable list? That way there are at least some options in the list

Reply:
I don't like the idea of truncating, because of this particular type of user: They won't type in the input, and instead manually scroll the list looking for the option they want. I've encountered this at work, no matter how nice you make the filtering features, their first instinct is to manually scroll until they find the option they want.

Ideally, we would implement infinite scroll here, so that they could scroll until they found the option they needed. I didn't want to tackle that in this issue.

Reply:
https://github.com/arrocke/gloss-translation/issues/222